### ISSUE TITLE
languages

### ISSUE THREAD

**Issue by cyrianR** [justchokingaround/jerry#72]
Hello, it may be a skill issue but I do not manage to make the option "-l french" work. Every time I try to watch an anime it is dubbed in english regardless of the -l option I use.
I tried with several animes and languages but it is always english.

**Comment by justchokingaround** [comment]
yeah that's bc it only work with providers that provide soft subtitles, aka external subtitle files, namely only aniwave atm, but its scraper is broken and i haven't fixed it yet. if u have a suggestion for a french website i could scrape lmk and i'll see if i can add it

**Comment by cyrianR** [comment]
> yeah that's bc it only work with providers that provide soft subtitles, aka external subtitle files, namely only aniwave atm, but its scraper is broken and i haven't fixed it yet. if u have a suggestion for a french website i could scrape lmk and i'll see if i can add it

Oke, well i'm looking forward to this fix then. Also, i think neko-sama.fr may be scrapped, and maybe anime-sama.cam too. Those are well known anime website for french people. That would be really great if you manage to integrate them.

**Comment by justchokingaround** [comment]
on a side note any ideas where i can rewatch "les misterieuses cites d'or" ?

**Comment by cyrianR** [comment]
> on a side note any ideas where i can rewatch "les misterieuses cites d'or" ?

Well, it seems like you can watch all 4 seasons on the official website www.france.tv since it's quite old. I mean, if you want to watch in french of course.

**Comment by justchokingaround** [comment]
yeah i do want to watch it in french, but france.tv only has the shitty new remakes. anw, ill try to find something else its not a problem
